packages/lesswrong/themes/stylePiping.ts

```diff
@@ -635,12 +638,16 @@ export const commentBodyStyles = (theme: ThemeType, dontIncludePointerEvents?: b
       ...theme.typography.commentBlockquote,
       ...theme.typography.body2,
       ...theme.typography.commentStyle,
-      color: theme.palette.text.bannerAdOverlay,
+      [isIfAnyoneBuildsItFrontPage]: {
```

Comment:
I think we can actually fully remove a lot of these. Might be worth going through and checking where the contrast is good enough. 

If the contrast isn't good enough without this change, we should make it permanent, since the background colors are also mostly permanent changes.